recipes/sos-matlab/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sos-matlab" %}
+{% set version = "0.18.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 54bf4ddcea30d43a90fc8a7432f990aae98351b9f0de4dc64196424ea1de99b9
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - sos-notebook >=0.19.4
+    - scipy
+    - octave  # [unix]
```

Comment:
octave and matlab are not the same thing right, is a fallback on octave the exected behavior from your users' point of view?

Reply:
This is tricky. The language module supports both MATLAB and Octave because they are similar in syntax. However, when the user installs `sos-matlab`, I do not know if they want to use it with MATLAB or Octave. This recipe adds octave as dependency to make it easy for users to set up an Octave environment, but this is not necessary if the user has MATLAB.

Reply:
understood. If you wanted to, you could make a second output of this package be `sos-octave` which just depends on `sos-matlab` and `octave`. Your choice really: I'm proposing something like this for opencv:
https://github.com/conda-forge/opencv-feedstock/pull/185/files#diff-e178b687b10a71a3348107ae3154e44cR176

Your changes would be:
```
outputs:
  - name: sos-matlab
  - name: sos-octave
    requirements:
      run:
        - {{ pin_subpackage('sos-matlab', exact=True) }}
        - octave  
        - octave_kernel
```

You probably don't even need to add the ` # [unix]` qualifier. The `sos-octave` package would just fail to install on platforms that don't have octave.

Reply:
Just did it, not sure if it works. I moved `octave` and `octave_kernel` to the `output`/`sos-octave` section so that they are not required for `sos-matlab`.